src/main/java/name/nkid00/teaminess/Teaminess.java

```diff
@@ -40,6 +39,12 @@ public class MinimalTp implements ModInitializer {
     public static final Style MSG_STYLE = Style.EMPTY.withColor(Formatting.YELLOW);
     public static final Style ACCEPT_STYLE = Style.EMPTY.withColor(Formatting.GREEN);
     public static final Style REFUSE_STYLE = Style.EMPTY.withColor(Formatting.RED);
+    public static final Formatting[] XAERO_COLORMAP = {
```

Comment:
Is this map xaero exclusive or vanilla compatible?

Perhaps we can use the map built in vanilla.